hw/ip/otbn/dv/uvm/gen-binaries.py

```diff
@@ -163,11 +132,7 @@ def main() -> int:
     script_dir = os.path.dirname(__file__)
     otbn_dir = os.path.normpath(os.path.join(script_dir, '../' * 2))
 
-    try:
-        toolchain = read_toolchain(args.obj_dir, otbn_dir)
-    except RuntimeError as err:
-        print(err, file=sys.stderr)
-        return 1
+    toolchain = read_toolchain(otbn_dir)
```

Comment:
This makes error reporting a bit ugly if there's a problem finding the toolchain: maybe worth leaving the `try / except` in?

Reply:
Fixed.